### DISCUSSION TITLE
integrate mozillas tls configurations

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#2916] [Ideas] [answered]
in reverse proxy guide and caddy/aache configs

**Comment by szaimen** [comment]
I thought Caddy would handle this by itself?

As for Apache, a PR would be welcome :)

**Reply by Zoey2936** [reply]
if i open the mozilla ssl (tls) page they show a custom config for caddy

**Reply by szaimen** [reply]
oh really? Shouldnt this then get reported to caddy so that they can adjust it in the project itself?

**Reply by Zoey2936** [reply]
maybe

**Comment by szaimen** [comment]
I found this regarding caddy:

![image](https://github.com/nextcloud/all-in-one/assets/42591237/04ad6968-a325-4b9a-97e2-950d4f333bf4)

Based on that I dont think we need it for caddy as we dont want to support IE anyway?

**Comment by szaimen** [comment]
However for the Apache RP docs it makes sense 👍

A PR would be welcome :)

**Reply by Zoey2936** [reply, marked as answer]
#3071
